### PR TITLE
Contribution: Education - Translation - GER - Nodes "Governance - V, U, W"

### PR DESCRIPTION
**Pull request by BicMC22**

### CODE REVIEW

**Comment by kaplan40** [comment]
If you correct the overlapping areas, I will combine them immediately.

**Comment by BicMC22** [comment]
> If you correct the overlapping areas, I will combine them immediately.
AW: Which areas overlap?

**Comment by DaAlien81** [comment]
ich werde kucken das ich das so schnell wie moglich erledige.

I will have a look if i can do this asap.

thx

Op ma 1 apr 2024 om 15:47 schreef BicMC22 ***@***.***>:

File: Projects/Governance/Schemas/Docs-Nodes/V/Votes/Votes-And-Claims-Programs/votes-and-claims-programs.json
Comment (BicMC22) [inline Projects/Governance/Schemas/Docs-Nodes/V/Votes/Votes-And-Claims-Programs/votes-and-claims-programs.json:1]: I have deleted the file. This was confirmed by the system, but the file still exists. Please use the files that are OK. I would like to submit the correct files for the reward this month. The table for translations has been adjusted.

File: Projects/Governance/Schemas/Docs-Nodes/G/Github/Github-Program/github-program.json
Comment (BicMC22) [inline Projects/Governance/Schemas/Docs-Nodes/G/Github/Github-Program/github-program.json:1]: I have deleted the file. This was confirmed by the system, but the file still exists. Please use the files that are OK. I would like to submit the correct files for the reward this month. The table for translations has been adjusted.

File: Projects/Governance/Schemas/Docs-Nodes/G/Governance/Governance-Project/governance-project.json
Comment (BicMC22) [inline Projects/Governance/Schemas/Docs-Nodes/G/Governance/Governance-Project/governance-project.json:1]: I have deleted the file. This was confirmed by the system, but the file still exists. Please use the files that are OK. I would like to submit the correct files for the reward this month. The table for translations has been adjusted.

**Comment by BicMC22** [comment]
I have deleted the file. This was confirmed by the system, but the file still exists. Please use the files that are OK. I would like to submit the correct files for the reward this month. The table for translations has been adjusted.

**Pull request closed without merging**